Makefile

```diff
@@ -25,23 +25,29 @@ TARGETS = \
 
 GINKGO ?= build/_output/bin/ginkgo
 
-$(GINKGO): Gopkg.toml
-	GOBIN=$$(pwd)/build/_output/bin/ go install ./vendor/github.com/onsi/ginkgo/ginkgo
+$(GINKGO): go.sum
+	GO111MODULE=on GOBIN=$$(pwd)/build/_output/bin/ go install $(GOPATH)/pkg/mod/github.com/onsi/ginkgo@v1.11.0/ginkgo
+
+
+
 
 # Make does not offer a recursive wildcard function, so here's one:
 rwildcard=$(wildcard $1$2) $(foreach d,$(wildcard $1*),$(call rwildcard,$d/,$2))
 
 # Gather needed source files and directories to create target dependencies
-directories := $(filter-out ./ ./vendor/ ,$(sort $(dir $(wildcard ./*/))))
```

Comment:
do you need to re-add vendor here?

Reply:
filter-out removes it. we need it as otherwise lots of files that were added y go mod vendor would end up as build  targets